txtdirect.go

```diff
@@ -175,16 +179,25 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 		return fmt.Errorf("option disabled")
 	}
 
-	if path == "/" {
-		http.Redirect(w, r, rec.Default, rec.Code)
-		return nil
-	}
+	fallback, code := getBaseTarget(rec)
 
-	if rec.Type == "path" && path != "" {
-		zone, from, _ := zoneFromPath(host, path)
-		rec, err = getFinalRecord(zone, from)
-		if err != nil {
-			return err
+	if rec.Type == "path" {
+		if path == "/" {
+			http.Redirect(w, r, rec.Default, rec.Code)
```

Comment:
If we agree on using rec.Root instead of rec.Default this needs a change here.

Reply:
From our Basecamp discussion, I think @stp-ip wanted to rename it, not have both